### PR TITLE
defines a new job that will run e2e perf encryption tests for kas-o

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

**Comment by p0lyn0mial** [comment]
requires https://github.com/openshift/cluster-kube-apiserver-operator/pull/667

**Comment by p0lyn0mial** [comment]
/hold

the required PR hasn't been merged.

File: ci-operator/jobs/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -212,6 +212,78 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws-operator-encryption,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
```
Comment (sttts) [inline ci-operator/jobs/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master-presubmits.yaml]: also 4.3?

**Pull request merged**